### DISCUSSION TITLE
Container SSH

### DISCUSSION THREAD

**Discussion by RFpersona** [okd-project/okd#1699] [Q&A]
I have a container in a pod, running on one of the compute nodes.  I would like to "ssh localhost", ie to the container itself.  It has the necessary packages for this, but I keep getting a "connection reset by localhost".  My question is, does OKD even support this?

**Comment by peterroth** [comment]
If you have a service (`oc -n <namespace> get svc`) for the SSH server running in the container, you can SSH into the pod. The best is if you have a route (`oc -n <namespace> get routes`) as well, then you can simply define hostname for the SSH server and access it from outside of the cluster.
If you want to access the container to run a few commands (f. g. you don't really need SSH), you can get into the container with `oc -n <namespace> rsh <podname> -c <containername>`.
